tests/utils.go

```diff
@@ -1437,6 +1437,16 @@ func DeletePV(os string) {
 	}
 }
 
+func DeleteVMI(vmi *v1.VirtualMachineInstance) {
+	if vmi != nil {
+		virtCli, err := kubecli.GetKubevirtClient()
+		PanicOnError(err)
+		_, err = virtCli.VirtualMachineInstance(NamespaceTestDefault).Get(vmi.Name, &metav1.GetOptions{})
+		Expect(errors.IsNotFound(err)).To(BeFalse())
```

Comment:
Could you please add Expect error message with the `err` and VMI name?

Reply:
I am not sure what you meant here, but did forget the actual delete...
I added it now, is that what you meant?
if the delete will fail it will include the error and the VMI name.

Reply:
This will panic with nil pointer dereference if you run IsNotFound on after Get succeeds. Consider using `And(HaveOccurred(), WithTransform(errors.IsNotFound, BeTrue())`

Reply:
Please see this comment  https://github.com/kubevirt/kubevirt/pull/3479/files#r468468287

Reply:
> This will panic with nil pointer dereference if you run IsNotFound on after Get succeeds. Consider using `And(HaveOccurred(), WithTransform(errors.IsNotFound, BeTrue())`

Oh , Thanks!
Done!

Reply:
> Please see this comment https://github.com/kubevirt/kubevirt/pull/3479/files#r468468287

When the tests fails it prints the error object which includes the VMI name and the error message,
since that I think there is no need to include it in the error formatting.( I think there is a discussion about this subject in this PR. I will attach it.) 
So I just added the "should successfully do something" part.
Is that acceptable?